### DISCUSSION TITLE
Verify the contract after dyployment Help needed /Thank You !

### DISCUSSION THREAD

**Discussion by DCVglobalnetwork** [Cyfrin/foundry-full-course-cu#2830] [Q&A]
![image](https://github.com/user-attachments/assets/2c6d8ace-df08-47ec-b1c3-915b91e6716b)

Hi I need a help as i am really getting tried of trying to verify each contract and no progress . Therefore I chose very basic and easy contract to do so . 

Counter-contract Foundry project  ,
```sh 
forge init
```

 deployed on sepolia and use ethereum API Key 
![image](https://github.com/user-attachments/assets/30ecbd84-5847-4325-bed1-ac0bd774d262)

And than on etherscan  testnet sepolia, nothing happened, no verification happened 

Please help

**Comment by EngrPips** [comment]
Hello @DCVglobalnetwork, Did you try `verifying` the contract during the deployment process? That is how I have actually done every verification step for a contract with Foundry, and it seems to always work.

**Reply by DCVglobalnetwork** [reply]
No, I haven't deployed and verified at the same time. I will do it, as I think with Foundry it is better to do it simultaneously. I will let you know. Thank you for getting back quickly

**Reply by EngrPips** [reply]
You are welcome. I am looking forward to hearing from you.
